### PR TITLE
labels: add quic label

### PR DESCRIPTION
**Pull request by jasnell**

### CODE REVIEW

File: lib/node-labels.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ const subSystemLabelsMap = new Map([
   [/^src\/node_report/, ['c++', 'report']],
   [/^src\/node_wasi/, ['c++', 'wasi']],
   [/^src\/node_worker/, ['c++', 'worker']],
+  [/^src\/quic\/*/, ['quic']],
+  [/^src\/node_bob*/, ['quic']],
```
Comment (richardlau) [inline lib/node-labels.js:37]:
```suggestion
  [/^src\/quic\/*/, ['c++', 'quic']],
  [/^src\/node_bob*/, ['c++', 'quic']],
```

File: lib/node-labels.js [hunk 2]
```diff
@@ -78,6 +80,8 @@ const subSystemLabelsMap = new Map([
   [/^deps\/uvwasi\//, 'wasi'],
   [/^deps\/nghttp2\/nghttp2\.gyp/, ['build', 'http2', 'dont-land-on-v6.x']],
   [/^deps\/nghttp2\//, ['http2', 'dont-land-on-v6.x']],
+  [/^deps\/ngtcp2\//, ['quic'],
+  [/^deps\/nghttp3\//, ['quic'],
```
Comment (richardlau) [inline lib/node-labels.js:84]:
```suggestion
  [/^deps\/ngtcp2\//, 'quic'],
  [/^deps\/nghttp3\//, 'quic'],
```

File: lib/node-labels.js [hunk 3]
```diff
@@ -133,6 +138,8 @@ const exclusiveLabelsMap = new Map([
   // n-api is treated separately since it is not a JS core module but is still
   // considered a subsystem of sorts
   [/^doc\/api\/n-api.md$/, ['doc', 'n-api']],
+  // quic
+  [/^doc\/api\/quic.md%/, ['quic']],
```
Comment (richardlau) [inline lib/node-labels.js:142]:
```suggestion
  [/^doc\/api\/quic.md$/, ['doc', 'quic']],
```

File: lib/node-labels.js (hunk 2 above)
Comment (phillipj) [inline lib/node-labels.js:84]: Seeing the syntax error reported from Travis, could this be the culprit?

Looks like the ones above either has one more closing bracket or uses a plain `string`..

File: lib/node-labels.js (hunk 3 above)
Comment (phillipj) [inline lib/node-labels.js]: Should we add the `doc` label as well as `quic` when the PR is related to docs?

Refs the other definitions having `doc` as the first item in the array in addition to whatever subsystem they relate to.

**Comment by phillipj** [comment]
It's alive in production 🚀

**Pull request merged**